content/docs/04-clusters/03-edge/04-edgeforge-workflow/03.5-palette-canvos.md

```diff
@@ -0,0 +1,658 @@
+<br />
+
+* **Provider OS image** - This is a [Kairos](https://kairos.io/)-based image. This image installs an immutable Operating System (OS) and software dependencies compatible with a specific Kubernetes version at runtime, i.e., during the cluster deployment.  
+
+
+In this guide, you will use Spectro Cloud's utility, [CanvOS](https://github.com/spectrocloud/CanvOS), to create an Edge installer ISO image and two provider OS images - one compatible with K3s v1.24.6 and another compatible with K3s v1.25.2. It builds two provider OS images, so you can use either of them that matches the Kubernetes version in your cluster profile.
+
+
+The diagram below shows the high-level steps to build the artifacts and push to an image registry.
+
+
+
+![Overarching diagram showing the workflow in the current guide.](/tutorials/palette-canvos/clusters_edge_palette-canvos_artifacts.png)
+
+
+
+
+This guide presents two workflows - Basic and Advanced. 
+
+The basic workflow has minimal customizations and offers a quick start to build Edge artifacts. This workflow builds an Ubuntu based Edge installer ISO and provider OS images. It will also push the provider OS images to the default image registry, *ttl.sh*. 
+
+The advanced workflow uses more customization options. This workflow builds an OpenSuse based Edge installer ISO and provider OS images. It will push the provider OS images to your Docker Hub image registry. 
+
+You can follow either of the workflows below that suits your use case.
+<br /> <br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Basic" key="basic_create_artifacts">
+
+
+# Prerequisites
+To complete this basic guide, you will need the following items:
+<br/>
+
+* A physical or virtual Linux machine with **AMD64** (also known as **x86_64**) processor architecture. You can issue the following command in the terminal to check your processor architecture. 
+  <br/>
+
+  ```bash
+  uname -m
+  ```
+
+* Minimum hardware configuration of the Linux machine:
+  - 4 CPU
+  - 8 GB memory
+  - 50 GB storage
+ 
+
+* [Git](https://cli.github.com/manual/installation). You can ensure git installation by issuing the `git --version` command.
+
+
+* [Docker Engine](https://docs.docker.com/engine/install/) version 18.09.x or later. You can use the `docker --version` command to view the existing Docker version. You should have root-level or `sudo` privileges on your Linux machine to create privileged containers.   
+<br />
+
+  <WarningBox>
+
+  Issue the following command to check if you can create privileged containers on your Linux machine.
+  <br />
+
+  ```bash
+  docker run --privileged hello-world
+  ```
+  If the command above fails, create a `docker` user group and add the current user to it using the instructions available [here](https://docs.docker.com/engine/install/linux-postinstall/). Alternatively, if you do not want to create a user group, ensure you can issue Docker commands with `sudo` privileges on your Linux machine.
+
+  </WarningBox>
```

Comment:
We don't need this, we just need to tell them to use `sudo`. Keep the bullet you have but let's remove this warning.